python/pysmurf/client/command/smurf_command.py

```diff
@@ -32,9 +32,13 @@
 
 class SmurfCommandMixin(SmurfBase):
 
+    # PV cache.  Used to just caget or caput every reg but this is not
+    # efficient.  See
+    # https://cars9.uchicago.edu/software/python/pyepics3/advanced.html.
+    _pv_cache = {}
```

Comment:
This should go in the SmurfControl `__init__` function since it is not a static variable.

Reply:
Thanks for catching this.  Done!